### DISCUSSION TITLE
How to enable hyperlinks

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#14924] [How to]
Dear all,

We need to create links to other websites in the knowledge base, FAQ and tickets areas. When we do this using the link button (CTRL + K) the hipertextual link does not work. Is that possible? 

Thanks in advance and best regards.

Yeti version: 6.1.0

**Comment by vovpff** [comment]
@sistemasnow Let's check on git about purifier. This tools control what links can be used.

**Reply by rskrzypczak** [reply]
Yes,
Here is your answer: https://github.com/YetiForceCompany/YetiForceCRM/issues/14263

**Reply by sistemasnow** [reply]
Ok.

We thought that the answer applied only to email templates ("mailto").

So, after checking [#14263 ](https://github.com/YetiForceCompany/YetiForceCRM/issues/14263) the solution would be:

- public static $purifierAllowedDomains = false;
or
- updating to YetiForce 6.2

Is that correct?

Thanks so much.

**Reply by rskrzypczak** [reply]
No. It works the same in version 6.2.
if you set it to false, you will turn off the verification globally.
you can try to set the domains first.

**Reply by sistemasnow** [reply]
Ok. I got it.

The false setting it works for us. 

How can we set the domains?

Thanks again.
